src/stylesheets/colors.less

```diff
@@ -0,0 +1,12 @@
+@background: #F8F8F8;
+@toolbar: #768EBA;
+@footer : #E7E7E7;
+@primary: #768EBA;
+@secondary: #E7E7E7;
+@active: #28A3E1;
+@disabled: #B3CDE0;
+@placeholder-text: #8C8C8C;
+@error-text: #D9534F;
+@text-light: #FFFFFF;
+@text-dark: #000000;
+@link-text: #76849B;
```

Comment:
nice, tho like i reviewed in col's PR, i want this to be renamed to constants.less